subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -931,13 +931,13 @@ int json_send_to_cloud(cJSON *const request)
 		return -ENOMEM;
 	}
 
-	LOG_DBG("Created request: %s", msg_string);
-
 	struct nct_dc_data msg = {
 		.data.ptr = msg_string,
 		.data.len = strlen(msg_string)
 	};
 
+	LOG_DBG("Created request: %s (size: %d)", msg.data, msg.data.len);
```

Comment:
```suggestion
	LOG_DBG("Created request: %s (size: %d)", msg_string, msg.data.len);
```

or `(char *)msg.data.ptr`

Reply:
Ha, you beat me to a force push fixing that